var/spack/repos/builtin/packages/py-pyside/package.py

```diff
@@ -33,17 +33,23 @@ class PyPyside(PythonPackage):
         preferred=True,
     )
 
-    depends_on("cmake", type="build")
-
+    # to prevent error: 'PyTypeObject' {aka 'struct _typeobject'} has no member
+    # named 'tp_print'
+    depends_on("python@:3.8", type=("build", "run"))
     depends_on("py-setuptools", type="build")
     # in newer pip versions --install-option does not exist
     depends_on("py-pip@:23.0", type="build")
+    depends_on("cmake", type="build")
+
     depends_on("py-sphinx", type=("build", "run"))
     depends_on("py-sphinx@:3.5.0", type=("build", "run"), when="@:1.2.2")
     depends_on("qt@4.5:4.9")
```

Comment:
```suggestion
    depends_on("qt@4.6:4.8")
```
According to setup.py

Reply:
Didn't see that, thanks.